client/src/components/Itinerary2.js

```diff
@@ -0,0 +1,103 @@
+// import React, { Component } from "react";
+// import { GET_ITINERARY } from "../actions/types";
+// import axios from "axios";
+// import { connect } from "react-redux";
+// import { getItineraries } from "../actions/itineraryActions";
+// import PropTypes from "prop-types";
+
+class Itinerary extends Component {
+  state = {
+    itineraries: []
+  };
+
+  componentDidMount() {
+    // this.state.getItineraries()
+    // const {match: {params}} = this.props;
+
+    const city = this.props.match.params.city;
+    console.log(city);
+    axios.get(`/api/itineraries/${city}`).then(res => {
+      console.log(res);
+      this.setState({
+        itineraries: res.data
+      });
+    });
+  }
+
+  getItineraries = city => {
+    console.log("hello?");
+
+    axios
+      .get(`/api/itineraries/${city}`)
+      .then(res => {
+        console.log(res);
+        this.setState({
+          itineraries: res.data
+        });
+      })
+      .catch(err => this.setState(err));
+  };
+
+  render() {
+    const { itineraries } = this.state;
+    const itineraryList = itineraries.length ? (
```

Comment:
I would extract this conditional code outside of the template to make it easier to follow whats getting rendered.  Basically using an `if` and assigning the conditional chunk to a variable like `returnedItineraries` or something semantic.